_assets/js/netlify/topics/makeBodyContent.js

```diff
@@ -0,0 +1,14 @@
+const md = window.markdownit();
+md.options['html'] = true;
+md.options['linkify'] = true;
+const parser = new DOMParser();
+// This function takes our body content as input, turns the markdown into HTML,
+// then loops over the html elements
```

Comment:
I think this might be a case of [this anti-pattern](https://blog.codinghorror.com/coding-without-comments/)

Reply:
Ah. Yeah, I self-identify as a junior dev, so this tracks.

Reply:
anything that requires anything more is probably over-engineering 🤷